packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -33,7 +33,6 @@ export interface IPendingMessage {
 	content: string;
```

Comment:
What all states can an IPendingMessage be in ?
1. Submitted
2. Acked by Server
3. Not Submitted - i.e. IMessageToStash?
4. Sequence - i.e. ISavedMessage?